### PR TITLE
Implement the personal api

### LINKED ISSUE #215: RPC method personal_newAccount not supported.
If I set `testrpc` as the provider for `web3` 

```javascript
const TestRPC = require("ethereumjs-testrpc");
const web3 = new Web3(TestRPC.provider());
```

and I try to create a new account:

```javascript
web3.personal.newAccount(password, (error, address) => {
        
});
```

I get the next error:

> RPC method personal_newAccount not supported.

If this a current limitation of `testrpc`? If that so, are you planning to give support to expose this API thought `testrpc`?

Thanks.

### PR DESCRIPTION
**Pull request by axic**
Based on https://github.com/ethereum/go-ethereum/wiki/Management-APIs#personal

Fixes #215 and #236.

### CODE REVIEW

**Comment by axic** [comment]
This is enough to work with browser-solidity and soltest (of solidity).

**Comment by tcoulter** [comment]
Any chance I can get some added tests for these methods?

File: test/requests.js [hunk 1]
```diff
@@ -838,6 +843,60 @@ var tests = function(web3) {
...
+    it("should return more than 0 accounts", function(done) {
+      web3.personal.getListAccounts(function(err, result) {
+        if (err) return done(err);
+        assert.equal(result.length, 2);
+        done();
+      });
+    });
+  });
+
+  describe("personal_unlockAccount", function() {
```
Comment (tcoulter) [inline test/requests.js:867]: This test could use more code that ensures the locked state of this account actually changed, vs. simply checking the boolean (i.e., make a transaction on the locked account, assert it doesn't work, unlock, make a transaction on the now unlocked account, assert it worked). Would be great if you could add this, but I'll do so if I can get to it.

Comment (axic) [reply test/requests.js:867]: @tcoulter I will not have time for a week at least, so please if you can, add those tests 😉

**Comment by tcoulter** [comment]
@axic Sorry for such the long wait on this. I've added it to the `release` branch of trufflesuite/ganache-core. Will go out with the next release of the TestRPC.

**Pull request closed without merging**